addon/ng2/blueprints/ng2/files/karma-test-shim.js

```diff
@@ -20,7 +20,7 @@ System.config({
   }
 });
 
-System.import('angular2/src/core/dom/browser_adapter').then(function(browser_adapter) {
+System.import('angular2/src/platform/browser/browser_adapter').then(function(browser_adapter) {
```

Comment:
tried to use just `angular2/platform/browser/browser_adapter` but it didn't work.